### PR TITLE
shantell-sans: init at 1.008

### LINKED ISSUE #231054: Font request: Shantell Sans
**Shantell Sans Font**

Shantell Sans, from [Shantell Martin](https://shantellmartin.art/), is a marker-style font built for creative expression, typographic play, and animation.

preview

**Metadata**

* homepage URL: https://shantellsans.com/
* source URL: https://github.com/arrowtype/shantell-sans
* license: ofl
* platforms: all

### PR DESCRIPTION
**Pull request by avanderbergh**
###### Description of changes

Shantell Sans, from [Shantell Martin](https://shantellmartin.art/), is a marker-style font built for creative expression, typographic play, and animation.
https://shantellsans.com/

###### Things done

- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

Fixes #231054

### CODE REVIEW

File: pkgs/data/fonts/shantell-sans/default.nix [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchzip,
+}:
```
Comment (JulienMalka) [inline pkgs/data/fonts/shantell-sans/default.nix:5]:
```suggestion
{ lib
, stdenvNoCC
, fetchzip
,
 }:
```

File: pkgs/data/fonts/shantell-sans/default.nix [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    mv *.ttf $out/share/fonts/truetype
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "Shantell Sans, from Shantell Martin, is a marker-style font built for creative expression, typographic play, and animation.";
+    homepage = "https://github.com/arrowtype/shantell-sans";
+    license = licenses.ofl;
+    maintainers = with maintainers; [avanderbergh];
```
Comment (JulienMalka) [inline pkgs/data/fonts/shantell-sans/default.nix:32]:
```suggestion
    maintainers = with maintainers; [ avanderbergh ];
```

File: pkgs/data/fonts/shantell-sans/default.nix [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    cd Shantell\ Sans\ ${version}/Desktop
+
+    mv Static $out/share/fonts/opentype
+    mv *.ttf $out/share/fonts/truetype
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "Shantell Sans, from Shantell Martin, is a marker-style font built for creative expression, typographic play, and animation.";
```
Comment (JulienMalka) [inline pkgs/data/fonts/shantell-sans/default.nix:29]:
```suggestion
    description = "A marker-style font built for creative expression, typographic play, and animation.";
```

**Review by JulienMalka — CHANGES_REQUESTED** [review]
Can you separate your changes in two commits:
- One where you add yourself as maintainer
- One where you add the package

**Comment by avanderbergh** [comment]
> Can you separate your changes in two commits:
> 
> * One where you add yourself as maintainer
> * One where you add the package

Done 👍🏼 @JulienMalka

**Comment by avanderbergh** [comment]
Could someone approve this?😅

**Comment by avanderbergh** [comment]
Hello? Can someone approve this? Or shall I just close pull request?

**Comment by avanderbergh** [comment]
🤷 Oh well... A year has passed and still no answer...

**Comment by tomodachi94** [comment]
If you're still interested in getting this merged, I'd be happy to take a look if you undelete the branch and reopen the PR. This project gets hundreds of contributions per day, and it's easy for some PRs to get lost.

(Advice on getting a committer to look at your work: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#i-opened-a-pr-how-do-i-get-it-merged)

**Pull request closed without merging**